set.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"github.com/codegangsta/cli"
+)
+
+var setCommand = cli.Command{
+	Name:  "set",
+	Usage: "set cgroup resources",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "config-file, c",
```

Comment:
This shouldn't require the path to config.json then. We probably want the bundlePath from the other PR.